lib/metaclient/meta_client.go

```diff
@@ -187,7 +187,7 @@ type MetaClient interface {
 	Users() []meta2.UserInfo
 	MarkDatabaseDelete(name string) error
 	MarkRetentionPolicyDelete(database, name string) error
-	MarkMeasurementDelete(database, mst string) error
+	MarkMeasurementDelete(database, mst, rpName string) error
```

Comment:
propose change the order of the parameters to make it align with other functions(like `Measurement` below).

```suggestion
	MarkMeasurementDelete(database, rpName, mst string) error
```